runtime/onert/core/src/compiler/Fp32ToFp16Converter.cc

```diff
@@ -222,7 +222,19 @@ void Fp32ToFp16Converter::appendNewOpSeqForConvertFp32ToFp16(const ir::OpSequenc
 {
   // OpSeq's input set is included in the first operation's input set
   const ir::OperandIndexSequence op_seq_inputs = op_seq.getInputs(); // copied
-  for (const auto &op_seq_input_ind : op_seq_inputs)
+
+  // Remove duplicated input index
+  // NOTE Please do not change sequence of op_seq_inputs. It can change the sequence of inputs of
+  // Subgraph
+  ir::OperandIndexSequence unique_op_seq_inputs;
+  for (const auto &ind : op_seq_inputs)
+  {
+    if (!unique_op_seq_inputs.contains(ind))
+    {
+      unique_op_seq_inputs.append(ind);
+    }
+  }
+  for (const auto &op_seq_input_ind : unique_op_seq_inputs)
```

Comment:
@ragmani I know nothing about `Fp32ToFp16`. However, it seems the changed code looks more complex.

Reply:
The changes prevent duplicated calling and creating unnecessary converting node for the same inputs.